spec/features/form/super_admin_view_forms_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+feature "View forms", type: :feature do
+  let(:org_forms) { [build(:form, id: 1, name: "Org form")] }
+  let(:other_org) { create :organisation, id: 2, slug: "Other org" }
+  let(:other_org_user) { create :user, organisation: other_org }
+  let(:other_org_forms) { [build(:form, id: 2, organisation_id: other_org.id, name: "Other org form")] }
+  # let(:other_org_forms) { [build (:form, id: 1, name: "Apply for a juggling license", organisation_id: other_org.id)] }
```

Comment:
Is this commented out line still needed?

Reply:
no! good spot I'll get rid

Reply:
it's gone 👍